### DISCUSSION TITLE
[Feature] "Open/Edit With..." button for local photos on mobile

### DISCUSSION THREAD

**Discussion by alecdotdev** [immich-app/immich#7612] [Feature Request]
### The feature

I would like the option to be able to quickly open/edit it in other apps like Google Photos instead of only the share option.
Similar to other gallery apps, shown below in Simple Gallery:

![image](https://github.com/immich-app/immich/assets/18179415/62463350-418f-444d-bad3-890c8a958cc9)

Currently if you want to crop a picture you're looking at in Immich, you'll have to just find it in another app to be able to edit. I think being able to trigger an edit in-app would be a time saver.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by AZweimiller** [comment]
Feels like a dupe of https://github.com/immich-app/immich/discussions/1605

**Reply by bo0tzz** [reply]
I think this is the other way around, taking a photo from Immich to another app.

**Comment by alecdotdev** [comment]
I attempted to fork [file_edit_launcher](https://github.com/enfold/file_edit_launcher) and use it to launch the edit intent and it worked, but there were issues with apps parsing the file path to the photo. It worked in a handful of editor-only apps but not Google Photos or Simple Gallery last time I tested it.

**Comment by bo0tzz** [comment]
This is technically older, but #13415 is more active so I'm closing this thread in favour of that.
